README.md

```diff
@@ -253,25 +262,28 @@ jobs:
     runs-on: ubuntu-latest
     name: job1
     steps:
-    - name: start minikube
-      uses: medyagh/setup-minikube@latest
-      id: minikube
-      with:
-        minikube-version: 1.24.0
-        driver: docker
-        container-runtime: containerd
-        kubernetes-version: v1.22.3
-        cpus: 4
-        memory: 4000m
-        cni: bridge
-        addons: registry,ingress
-        extra-config: 'kubelet.max-pods=10'
-        feature-gates: 'DownwardAPIHugePages=true'
-        mount-path: '/Users/user1/test-files:/testdata'
-        insecure-registry: localhost:5000,10.0.0.0/24
-    # now you can run kubectl to see the pods in the cluster
-    - name: kubectl
-      run: kubectl get pods -A
+      - name: start minikube
+        uses: medyagh/setup-minikube@latest
+        id: minikube
+        with:
+          cache: false
+          minikube-version: 1.24.0
+          driver: docker
+          container-runtime: containerd
+          kubernetes-version: v1.22.3
+          cpus: 4
+          memory: 4000m
+          cni: bridge
+          addons: registry,ingress
+          extra-config: 'kubelet.max-pods=10'
+          feature-gates: 'DownwardAPIHugePages=true'
+          mount-path: '/Users/user1/test-files:/testdata'
+          wait: false
+          insecure-registry: 'localhost:5000,10.0.0.0/24'
+          start-args: '--delete-on-failure --network mynetwork'
```

Comment:
lets use a flag that wont break if ppl copy paste, 
(not sure if --network works this way or not) instead of --network maybe use something else?

Reply:
Good call, fixed